Formula/varnish.rb

```diff
@@ -61,5 +61,18 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{sbin}/varnishd -V 2>&1")
+
+    # extract a subset of the varnishtest tests:
+    # - b*.vtc (basic functionality)
+    # - m*.vtc (VMOD modules, including loading), but skipping m00000.vtc which is known to fail
+    #   but is "nothing of concern" (see varnishcache/varnish-cache#3710)
+    # - u*.vtc (utilities and background processes)
+    system "tar", "-xzf", cached_download, "-C", testpath,
+        "--strip-components=4", "--exclude", "m00000.vtc",
+        "varnish-*/bin/varnishtest/tests/[bmu]*.vtc"
```

Comment:
Since we're only running a small subset of the tests, would it be better to just install those into the bottle during the build (perhaps in `pkgshare`) and then run those tests from there?

Reply:
Sure, I didn't know that was an option. Would we put _all_ tests in `#{pkgshare}/tests`?

It would make sense, I think; I certainly would appreciate having access to `./bin/varnishtest/tests` and `./vmod/tests` test files being available at `#{pkgshare}/tests` and `#{pkgshare}/tests/vmod`, for example.